torch/CMakeLists.txt

```diff
@@ -310,7 +310,7 @@ endif()
 
 add_library(torch_python SHARED ${TORCH_PYTHON_SRCS})
 torch_compile_options(torch_python)  # see cmake/public/utils.cmake
-if(NOT WIN32)
+if(APPLE)
```

Comment:
Why this check here? Imo symbol visibility on Linux and Mac should be identical, shouldn't it?

Reply:
Apple has some hard to find linking errors that lead to test failures